addons/mrp/mrp_demo.xml

```diff
@@ -690,6 +690,7 @@
 
     <record id="mrp_production_1" model="mrp.production">
         <field name="product_id" ref="product.product_product_3"/>
+        <field name="categ_id" ref="product.product_category_5"/>
```

Comment:
In old api related field is not set automatically. so need to caeg_id in demo data